api/test/shell/cli_test.sh

```diff
@@ -34,29 +47,47 @@ go build -o ./manager-api .
 sleep 3
 pkill -f manager-api
 
-if [[ ! -f "./logs/error.log" ]]; then
-    echo "failed: failed to write log"
+check_logfile
+
+if [[ `grep -c "INFO" ${logfile}` -ne '0' ]]; then
+    echo "failed: should not write info log when level is warn"
     exit 1
 fi
 
-if [[ `grep -c "INFO" ./logs/error.log` -ne '0' ]]; then
-    echo "failed: should not write info log when level is warn"
+clean_logfile
+
+# change level and test signal
+
+sed -i 's/warn/info/' conf/conf.yaml
```

Comment:
I think that is unsafe, should match and replace `level: warn`

Reply:
That's right. Agree with you.